pages/[...slug].vue

```diff
@@ -1,12 +1,34 @@
 <template>
   <div>
-    <Breadcrumbs v-if="page.breadcrumbs" />
-    <DrupalTabs v-if="page.local_tasks" :tabs="page.local_tasks" />
-    <component :is="renderCustomElements(page.content)" />
+    <NuxtLayout :name="layout">
+      <main>
+        <SiteBreadcrumbs />
+        <DrupalTabs v-if="page.local_tasks" :tabs="page.local_tasks" />
+        <component :is="renderCustomElements(page.content)" />
+      </main>
+    </NuxtLayout>
   </div>
 </template>
 
 <script lang="ts" setup>
-const { fetchPage, renderCustomElements } = useDrupalCe()
-const page = await fetchPage(useRoute().path, { query: useRoute().query })
+const { fetchPage, renderCustomElements } = useDrupalCe();
+const page = await fetchPage(useRoute().path, { query: useRoute().query });
+// Set to false to support custom layouts, using <NuxtLayout> instead.
+definePageMeta({
+  layout: false,
+});
+const layout = computed(() => {
+  return page.value.page_layout || "default";
```

Comment:
this is quite verbose and not nice to see here. I think it would be nice to shorten this, e.g. by introducing a helper in our composable. What do you think about opening an issue for nuxt-drupal-ce and adding it to make this nicer?

Reply:
will create a follow-up ticket to add this in nuxt-drupal-ce